<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>CPropertyEditor</class>
  <widget class="QWidget" name="CPropertyEditor">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>777</width>
        <height>500</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Dependent property editor</string>
    </property>
    <property name="windowIcon">
      <iconset resource="../../DyssolGUI/Resources.qrc">
        <normaloff>:/Icons/WinIcon</normaloff>:/Icons/WinIcon</iconset>
    </property>
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,1,0" columnstretch="1,0">
      <property name="leftMargin">
        <number>0</number>
      </property>
      <property name="topMargin">
        <number>0</number>
      </property>
      <property name="rightMargin">
        <number>0</number>
      </property>
      <property name="bottomMargin">
        <number>0</number>
      </property>
      <item row="2" column="1" rowspan="2">
        <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
            <string>Calculator</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
              <widget class="QLabel" name="label_4">
                <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
                </property>
                <property name="text">
                  <string>T [K]:</string>
                </property>
              <property name="toolTip"><string>Temperature value</string></property><property name="whatsThis"><string>Temperature value</string></property></widget>
            </item>
            <item>
              <widget class="QLineEdit" name="lineCalcT">
                <property name="toolTip">
                  <string>Temperature value</string>
                </property>
              <property name="whatsThis"><string>Temperature value</string></property></widget>
            </item>
            <item>
              <widget class="QLabel" name="label_5">
                <property name="text">
                  <string>P [Pa]:</string>
                </property>
              <property name="toolTip"><string>Pressure value</string></property><property name="whatsThis"><string>Pressure value</string></property></widget>
            </item>
            <item>
              <widget class="QLineEdit" name="lineCalcP">
                <property name="toolTip">
                  <string>Pressure value</string>
                </property>
              <property name="whatsThis"><string>Pressure value</string></property></widget>
            </item>
            <item>
              <widget class="QLabel" name="labelPropertyName">
                <property name="text">
                  <string>Property:</string>
                </property>
              <property name="toolTip"><string>Calculated parameter value</string></property><property name="whatsThis"><string>Calculated parameter value</string></property></widget>
            </item>
            <item>
              <widget class="QLineEdit" name="lineCalcVal">
                <property name="toolTip">
                  <string>Calculated parameter value</string>
                </property>
                <property name="readOnly">
                  <bool>true</bool>
                </property>
              <property name="whatsThis"><string>Calculated parameter value</string></property></widget>
            </item>
            <item>
              <spacer name="verticalSpacer">
                <property name="orientation">
                  <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                  <size>
                    <width>20</width>
                    <height>337</height>
                  </size>
                </property>
              </spacer>
            </item>
          </layout>
          <property name="toolTip">
            <string>Property calculator</string>
          </property>
          
          <property name="whatsThis">
            <string>Calculation of property values by given parameters</string>
          </property>
        </widget>
      </item>
      <item row="2" column="0" rowspan="2">
        <widget class="QGroupBox" name="groupBox">
          <property name="title">
            <string>Correlation plot</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayoutPlot" stretch="0,1">
                <item>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                      <widget class="QRadioButton" name="radioButtonT">
                        <property name="text">
                          <string>Temperature</string>
                        </property>
                        <property name="checked">
                          <bool>true</bool>
                        </property>
                        <property name="toolTip">
                          <string>Parameter for X axis</string>
                        </property>
                        
                        <property name="whatsThis">
                          <string>Parameter for X axis</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QRadioButton" name="radioButtonP">
                        <property name="text">
                          <string>Pressure</string>
                        </property>
                        <property name="toolTip">
                          <string>Parameter for X axis</string>
                        </property>
                        
                        <property name="whatsThis">
                          <string>Parameter for X axis</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <spacer name="horizontalSpacer">
                        <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                        </property>
                        <property name="sizeHint" stdset="0">
                          <size>
                            <width>40</width>
                            <height>20</height>
                          </size>
                        </property>
                      </spacer>
                    </item>
                    <item>
                      <widget class="QLabel" name="labelPlotValue">
                        <property name="text">
                          <string>P [Pa]:</string>
                        </property>
                        <property name="toolTip">
                          <string>Value of the second correlation parameter</string>
                        </property>
                        
                        <property name="whatsThis">
                          <string>Value of the second correlation parameter</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QLineEdit" name="linePlotValue">
                        <property name="toolTip">
                          <string>Value of the second correlation parameter</string>
                        </property>
                        
                        <property name="whatsThis">
                          <string>Value of the second correlation parameter</string>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item>
                  <widget class="QtPlot::CQtPlot" name="plot" native="true">
                    <property name="toolTip">
                      <string>Correlation plot in 2D</string>
                    </property>
                    
                    <property name="whatsThis">
                      <string>Correlation plot in 2D</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        <property name="toolTip"><string>Visual representation of configured correlations</string></property><property name="whatsThis"><string>Visual representation of configured correlations</string></property></widget>
      </item>
      <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
            <string>Correlations</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
            <item>
              <widget class="CQtTable" name="listCorrelations">
                <property name="selectionMode">
                  <enum>QAbstractItemView::SingleSelection</enum>
                </property>
                <property name="verticalScrollMode">
                  <enum>QAbstractItemView::ScrollPerPixel</enum>
                </property>
                <property name="horizontalScrollMode">
                  <enum>QAbstractItemView::ScrollPerPixel</enum>
                </property>
                <attribute name="horizontalHeaderVisible">
                  <bool>false</bool>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                  <bool>true</bool>
                </attribute>
                <property name="toolTip">
                  <string>List of correlations defining the selected temperature-/pressure-dependent compound property</string>
                </property>
                
                <property name="whatsThis">
                  <string>List of correlations defining the selected temperature-/pressure-dependent compound property</string>
                </property>
              </widget>
            </item>
            <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                  <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                      <size>
                        <width>40</width>
                        <height>20</height>
                      </size>
                    </property>
                  </spacer>
                </item>
                <item>
                  <widget class="QPushButton" name="buttonAdd">
                    <property name="maximumSize">
                      <size>
                        <width>63</width>
                        <height>23</height>
                      </size>
                    </property>
                    <property name="font">
                      <font>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                      </font>
                    </property>
                    <property name="text">
                      <string>+</string>
                    </property>
                    <property name="toolTip">
                      <string>Add new correlation</string>
                    </property>
                    
                    <property name="whatsThis">
                      <string>Add new correlation</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="buttonRemove">
                    <property name="maximumSize">
                      <size>
                        <width>63</width>
                        <height>23</height>
                      </size>
                    </property>
                    <property name="font">
                      <font>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                      </font>
                    </property>
                    <property name="text">
                      <string>-</string>
                    </property>
                    <property name="toolTip">
                      <string>Remove selected correlation</string>
                    </property>
                    
                    <property name="whatsThis">
                      <string>Remove selected correlation</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="buttonUp">
                    <property name="maximumSize">
                      <size>
                        <width>63</width>
                        <height>23</height>
                      </size>
                    </property>
                    <property name="font">
                      <font>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                      </font>
                    </property>
                    <property name="text">
                      <string>▲</string>
                    </property>
                    <property name="toolTip">
                      <string>Move selected correlation upwards in the list</string>
                    </property>
                    
                    <property name="whatsThis">
                      <string>Move selected correlation upwards in the list</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="buttonDown">
                    <property name="maximumSize">
                      <size>
                        <width>63</width>
                        <height>23</height>
                      </size>
                    </property>
                    <property name="font">
                      <font>
                        <pointsize>10</pointsize>
                        <weight>75</weight>
                        <bold>true</bold>
                      </font>
                    </property>
                    <property name="text">
                      <string>▼</string>
                    </property>
                    <property name="toolTip">
                      <string>Move selected correlation downwards in the list</string>
                    </property>
                    
                    <property name="whatsThis">
                      <string>Move selected correlation downwards in the list</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <customwidgets>
    <customwidget>
      <class>CQtTable</class>
      <extends>QTableWidget</extends>
      <header>QtTable.h</header>
    </customwidget>
    <customwidget>
      <class>QtPlot::CQtPlot</class>
      <extends>QWidget</extends>
      <header>QtPlot.h</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>listCorrelations</tabstop>
    <tabstop>buttonAdd</tabstop>
    <tabstop>buttonRemove</tabstop>
    <tabstop>buttonUp</tabstop>
    <tabstop>buttonDown</tabstop>
    <tabstop>radioButtonT</tabstop>
    <tabstop>radioButtonP</tabstop>
    <tabstop>linePlotValue</tabstop>
    <tabstop>label_4</tabstop>
    <tabstop>lineCalcT</tabstop>
    <tabstop>lineCalcP</tabstop>
    <tabstop>lineCalcVal</tabstop>
  </tabstops>
  <resources>
    <include location="../../DyssolGUI/Resources.qrc" />
  </resources>
  <connections />
</ui>